addons/account/views/account_move_views.xml

```diff
@@ -1244,6 +1244,27 @@
                                                invisible="auto_post in ('no', 'at_date')"
                                                readonly="state != 'draft'"/>
                                         <field name="to_check"/>
+                                        <field name="system_exchange_rate" invisible="True"/>
+                                        <field name="exchange_rate" invisible="True"/>
+                                        <label for="display_rate" invisible="currency_id == company_currency_id or state != 'draft'"/>
```

Comment:
If I set the rate it might be good to see it even when posted?

Reply:
Yes we thought it might be a good idea too (there is a feedback on the task)

Basically, there are 2 challenges:
1. Custom Rate: You could change the debit/credit values of a tax line (giving the line a different currency rate)
2. System Rate: You change the days rate (res.currency) after posting the invoice. The rate displayed is the not the same as we did not store the system rate that was used.

The only way is to read the rate from the lines when the move is posted, but which line?

Reply:
Also the changes are logged in the chatter, so when a custom rate is used, it is recorded.

Reply:
I agree, AFAIK most softwares display it in a very visible manner.

But I guess that different people want different things and Odoo offers the possibility to customize that.
... if `system_exchange_rate` is always computed